src/runner.jl

```diff
@@ -223,26 +223,32 @@ function test!(pkg::AbstractString,
 
             sandbox(sandbox_args...) do
                 flush(stdout)
-                try
-                    @info "Testing $pkg"
-                    run(cmd)
+                @info "Testing $pkg"
+                proc = open(cmd, Base.stdout; write=true)
+                wait(proc)
+                if proc.exitcode == 0
                     @info "$pkg tests passed. Results saved to $logfilename."
-                catch err
+                elseif proc.exitcode == 2
```

Comment:
Maybe we should define a const global for this magic number?

Reply:
Have added it in as a constant